spi_nand_flash/idf_component.yml

```diff
@@ -0,0 +1,5 @@
+version: "0.0.3"
```

Comment:
I'm also not sure about the version. Why 0.0.3, specifically, rather than 0.1.0 or 0.0.1? Were there 2 other versions before?

Also, we may want to indicate somewhere the version of dhara library we are using. Or even better, make `dhara` a separate component with just the upstream library and nothing else, so that the version of `dhara` component can match the version of the upstream library. Then `spi_nand_flash` component can depend on `dhara` component.

Reply:
I'm in favor of making dhara a separate component.

Reply:
The purpose of adding dhara as a separate component is:
- other NAND Flash wrappers/drivers can also use the already-available dhara component, instead of adding it as a submodule
- if a bugfix release of dhara component is made, users can get it without necessarily having to bump the version of spi_nand_flash